tasks/bootstrap-tasks.yml

```diff
@@ -35,6 +35,12 @@
   tags:
     - set-hostname
 
+- name: Set time zone
+  command: timedatectl set-timezone {{ ntp_timezone }}
+
+- name: Enable NTP
+  command: timedatectl set-ntp true
```

Comment:
It uses the default Debian pool from ntp.org which is defined at systemd compile-time.[1] In addition, the user can configure custom servers in `/etc/systemd/timesyncd.conf`.

```
[Time]
NTP=0.debian.pool.ntp.org 1.debian.pool.ntp.org 2.debian.pool.ntp.org 3.debian.pool.ntp.org
```
[1] https://sources.debian.net/src/systemd/215-17%2Bdeb8u6/debian/rules/#L28